src/vxingest/builder_common/vx_ingest.py

```diff
@@ -197,6 +202,25 @@ def get_file_list(self, df_query, directory, file_pattern, file_mask):
             result = self.cluster.query(df_query)
             df_elements = list(result)
             df_full_names = [element["url"] for element in df_elements]
+            # Handle if the directory is a URL or a local path
+            if str(directory).startswith("http://") or str(directory).startswith(
+                "https://"
+            ):
+                logger.warning(
+                    "get_file_list: Directory is a URL, skipping local file glob."
+                )
+                file_list = []
+            elif str(directory).startswith("s3://"):
+                logger.warning(
+                    "get_file_list: Directory is an S3 path, skipping local file glob."
+                )
+                file_list = []
+            elif str(directory).startswith("file://"):
+                # local file path with file:// prefix
+                self.load_spec["input_data_path"] = pathlib.Path(
+                    directory[7:]
+                ).as_posix()
+                directory = directory[7:]
             if pathlib.Path(directory).exists() and pathlib.Path(directory).is_dir():
```

Comment:
What happens if the directory doesn't have a URI prefix? That seems like a common error.